strip.lua

```diff
@@ -244,7 +247,7 @@ function mine()
         state.position.trunkFacing = 1
     end
 
-    if checkInv() == false and state.mine.index == 1 then
+    if ((checkInv() == false or checkFuel()) == false and state.mine.index == 1) then
```

Comment:
```suggestion
    if ((checkInv() or checkFuel()) == false and state.mine.index == 1) then
```
I think I made a poor suggestion :P